compiler/lib/javascript.ml

```diff
@@ -129,7 +129,10 @@ type identifier = string
 
 type ident_string =
   { name : identifier
-  ; var : Code.Var.t option }
+  ; var : Code.Var.t option
+  ; loc : location}
+
+let mk ?(loc=N) ?var name = { name ; var ; loc }
```

Comment:
The name `mk` looks a bit weird to me.
What about having a function `ident` instead, defined as
```
let ident ?(loc=N) ?var name = S { name ; var ; loc }
```

Reply:
Fixed.